src/amo/reducers/home.js

```diff
@@ -83,6 +83,7 @@ export type PrimaryHeroShelfType =
 
 export type HeroCallToActionType = {|
   url: string,
+  outgoing: string,
```

Comment:
Shouldn't the API return an `OutgoingUrl` object instead of inlining it?

Reply:
I thought about that while developing mozilla/addons-server#16125 - it seemed weird (to me) to have another `url` inside the `url`, an extra level of object, when cta was already an object, and for a similar reason.  It'd mean the data structure would be:
```json
results = {
    description: "baa",
    headline: "foo",
    cta: {
        url: {
            url: "https://awesome.site/",
            outoing: "https://outgoing.m.c/3423&awesome"
        },
        text: "go to site"
    }
}
"""

Reply:
I had thought about this too during my review, but decided that the way it's being handled here, via inline properties just makes more sense to me.